modules/color-picker/react/spec/ColorInput.spec.tsx

```diff
@@ -1,115 +1,150 @@
+        );
+        expect(getByTestId(id)).toHaveAttribute('placeholder', placeholder);
+      });
+    });
+
+    describe('with an id', () => {
+      test('should render a ColorInput with an id', () => {
+        const {getByTestId} = render(<ColorInput id="my-id" data-testid={id} />);
+        expect(getByTestId(id)).toHaveAttribute('id', 'my-id');
+      });
+    });
+
+    describe('with a value', () => {
+      test('should render a ColorInput with a value', () => {
+        const {getByTestId} = render(<ColorInput value={value} data-testid={id} />);
+        expect(getByTestId(id).value).toBe(value);
+      });
+    });
+
+    describe('with a value and check', () => {
+      test('should render a ColorInput with a value and check', () => {
+        const {container} = render(<ColorInput value={value} showCheck={true} />);
+
+        expect(container.querySelector('svg')).toHaveClass('wd-icon-check-small');
+      });
+    });
+
+    describe('with a value and background', () => {
+      test('should render a ColorInput with a value and the value as a background', () => {
+        const {container} = render(<ColorInput value={value} />);
+
+        expect(container.querySelector('div div div[class^="css"]')).toHaveStyle(
+          'background-color: #eee;'
+        );
+      });
+    });
+
+    describe('with disabled attribute', () => {
+      test('should render a disabled ColorInput', () => {
+        const {getByTestId} = render(<ColorInput disabled={true} data-testid={id} />);
+        expect(getByTestId(id)).toBeDisabled();
+      });
+    });
 
-  test('ColorInput should spread extra props', () => {
-    const component = mount(<ColorInput data-propspread="test" />);
-    const input = component
-      .find('input') // TODO: Standardize on prop spread location (see #150)
-      .getDOMNode();
-    expect(input.getAttribute('data-propspread')).toBe('test');
-    component.unmount();
+    describe('with extra, arbitrary props', () => {
+      test('should spread extra props', () => {
+        const attr = 'test';
+        const {getByTestId} = render(<ColorInput data-propspread={attr} data-testid={id} />);
+        expect(getByTestId(id)).toHaveAttribute('data-propspread', attr);
+      });
+    });
+
+    describe('when provided an input ref', () => {
+      test('input ref should be defined', () => {
+        const ref: React.RefObject<HTMLInputElement> = React.createRef();
+        render(<ColorInput inputRef={ref} />);
+        expect(ref.current).toBeDefined();
+      });
+    });
+
+    describe('when provided a value with a hashtag', () => {
+      test('the value is stripped', () => {
+        const {getByTestId} = render(<ColorInput value={'#eee'} data-testid={id} />);
+        expect(getByTestId(id).value).toBe(value);
+      });
+    });
+
+    describe('when provided more than 6 characters as the value', () => {
+      test('the value is stripped', () => {
```

Comment:
This looks like a copy and paste error from above? Something like `'the value is truncated to a length of 6'` or something like that